packages/core/src/JWEditor.ts

```diff
@@ -73,13 +77,13 @@ export class JWEditor {
         this.el.appendChild(this.editable);
         document.body.appendChild(this.el);
 
+        // Init the event manager now that the cloned editable is in the DOM.
+        this.eventManager = new EventManager(this);
+
```

Comment:
Why the move ? Just wondering.

Reply:
We need the event manager before loading a plugin because `_loadShortcut` need it